.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Node Modules Cache
         uses: actions/cache@v2
         with:
-          path: '**/node_modules'
+          path: "**/node_modules"
```

Comment:
It seems we use single quotes in this file. Why the change?

Reply:
reverted.